docs/Reference/Pantheon-API-Methods.md

```diff
@@ -3771,6 +3771,56 @@ None
     The `EEA` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `EEA` API methods.
 
+### eea_getPrivateTransaction
+
+Returns the private transaction if you are a participant; otherwise, null. Get the public transaction 
```

Comment:
Minor suggestion, maybe make this a new paragraph. 
"To get the public transaction, use.."

Reply:
Done